spring-cloud-starter-stream-processor-tensorflow/src/main/java/org/springframework/cloud/stream/app/tensorflow/processor/TensorFlowService.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2015-2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.stream.app.tensorflow.processor;
+
+import static org.apache.commons.io.IOUtils.buffer;
+import static org.apache.commons.io.IOUtils.toByteArray;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Map;
+import java.util.Map.Entry;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.tensorflow.Graph;
+import org.tensorflow.Session;
+import org.tensorflow.Session.Runner;
+import org.tensorflow.Tensor;
+
+import org.springframework.core.io.Resource;
+import org.springframework.tuple.Tuple;
+
+/**
+ * @author Christian Tzolov
+ */
+public class TensorFlowService implements AutoCloseable {
+
+	private static final Log logger = LogFactory.getLog(TensorflowProcessorConfiguration.class);
+
+	private Graph graph;
+
+	public TensorFlowService(Resource modelLocation) throws IOException {
+		try (InputStream is = modelLocation.getInputStream()) {
+			graph = new Graph();
+			logger.info("Loading TensorFlow graph model: " + modelLocation );
```

Comment:
Must be wrapped with the `logger.is InfoEnabled()` to avoid extra string concatenation operation for nothing.